solutions/bronze/cf-581D.mdx

```diff
@@ -89,4 +89,169 @@ int main() {
 ```
 
 </CPPSection>
+<JavaSection>
+
+```java
+import java.util.*;
+
+public class Main {
```

Comment:
bruh why'd you undo my class name change to `ThreeLogos`

Reply:
So you want me to change the class name to ThreeLogos right?